src/exchange_solver.cc

```diff
@@ -3,11 +3,25 @@
 #include <vector>
 #include <map>
 
+#include "context.h"
 #include "exchange_graph.h"
 
 namespace cyclus {
 
-double ExchangeSolver::PseudoCost(double cost_add) {
+double ExchangeSolver::Cost(const Arc& a, bool exclusive_orders) {
+  return (exclusive_orders && a.exclusive()) ?
```

Comment:
Does this mean that if the users turns of exclusive orders, then no exclusive orders are possible at all?  This will break the cycamore reactor.